submitit/slurm/slurm.py

```diff
@@ -409,6 +409,7 @@ def _make_sbatch_string(
     array_parallelism: int = 256,
     wckey: str = "submitit",
     map_count: tp.Optional[int] = None,  # used internally
+    qos: tp.Optional[str] = None, # quality of service
```

Comment:
Could you move this one line up , above "map_count" ?

Reply:
```suggestion
    qos: tp.Optional[str] = None, # quality of service
```